integration-tests/maven/src/test/resources/projects/parent-pom-it/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.acme.it</groupId>
+  <artifactId>acme-parent-pom</artifactId>
+  <version>0.0.1.BUILD-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <properties>
+    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.version>@project.version@</quarkus.platform.version>
+  </properties>
+  <dependencyManagement>
+    <dependencies>
+      <!-- insert managed dependencies here -->
+      <dependency>
+        <groupId>${quarkus.platform.group-id}</groupId>
+        <artifactId>${quarkus.platform.artifact-id}</artifactId>
+        <version>${quarkus.platform.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
```

Comment:
This import isn't really required, correct? The generated module will import whatever it does today. This kind of integration with the parent is the tricky part.

Reply:
I can change the parent pom to a completely empty xml containing only GAV and packaging.

Reply:
yes, that will probably make sense in this case. thanks!